src/relay/backend/graph_runtime_codegen.cc

```diff
@@ -419,7 +419,7 @@ class GraphRuntimeCodegen
     auto pf1 = GetPackedFunc("relay.backend._CompileEngineLower");
     Target target;
     // Handle external function
-    if (func->GetAttr<tir::StringImm>(attr::kCompiler).defined()) {
+    if (func->GetAttr<runtime::String>(attr::kCompiler).defined()) {
```

Comment:
Let us consider expose runtime::String to the tvm namespace as well(by put using runtime::String in the tvm namespace of container.h) So that we can directly type GetAttr<String>